lib/DTL/Fast/Filter/Default.pm

```diff
@@ -21,6 +21,10 @@ sub filter
 {
     my ($self, $filter_manager, $value, $context) = @_;
 
+    if ( !defined $value || ( $value eq '' ) ) {
```

Comment:
redundant parens, see https://perldoc.perl.org/perlop.html#Operator-Precedence-and-Associativity
also, `!$value` should be ok here.